src/webots/nodes/utils/WbUrl.cpp

```diff
@@ -38,6 +38,11 @@ const QString &WbUrl::missingTexture() {
   return missingTexture;
 }
 
+const QString &WbUrl::missingIcon() {
+  const static QString missingIcon = WbStandardPaths::resourcesPath() + "images/missing_icon.png";
+  return missingIcon;
+}
+
```

Comment:
I believe it is better to be more precise with naming:
```suggestion
const QString &WbUrl::missingProtoIcon() {
  const static QString missingProtoIcon = WbStandardPaths::resourcesPath() + "images/missing_proto_icon.png";
  return missingProtoIcon;
}

```